### PR TITLE
Added new reference: Eventarc-apihub

### PR DESCRIPTION
**Pull request by mat-613**
Added a new reference: Eventarc + ApiHub integration - A reference for a workflow-based automation for Apigee X using API Hub and EventArc based on a recent customer implementation scenario.

### CODE REVIEW

**Comment by danistrebel** [comment]
Ciao Matteo, thanks a lot for the example and the PR. Great work.

As of now, we don't have an API hub setup for the Apigee DevRel repository that would allow us to perform the required continuous testing via our testing framework.

As the main contribution of this article is in the fantastic instructions in the README as well as the workflow configs I was wondering if you're better suited to creating a blog post in the [Apigee Community Forum](https://www.googlecloudcommunity.com/gc/Apigee/bd-p/cloud-apigee) or by targeting the [Cloud Community on Medium](https://medium.com/google-cloud).

**Comment by OmidTahouri** [comment]
Hi Matteo -- thanks again for this contribution. Daniel has made valid points above but after recent discussions, let's try to make some progress on this PR. Please review the following points.

1. Do you see any challenges with submitting this project to be a Lab instead of a Reference? I feel that's more appropriate since it's a step-by-step guided setup of a solution.

2. Please do not remove the templated PR description that is automatically provided to you when creating the PR. You must complete each section of the template. For examples, see PRs 664, 637, 630.

3. Per [CONTRIBUTING.md](https://github.com/apigee/devrel/blob/main/CONTRIBUTING.md#pull-request-checks):
> Commit messages follow [Conventional Commits](https://www.conventionalcommits.org/) standards to tell the release automation which SemVer version increase is intended 

Please see the repo's commit history amd other PRs for reference. Consider squashing all of your commits/changes into one commit e.g. `feat: introducing eventarc apihub reference` (and `push -f` to your own branch)

4. A `pipeline.sh` file is required to be in the root of your project's directory. Please see other projects.

Given the impact of some of these changes, I suggest creating a new PR when you're ready, making sure you maintain the PR template. We're happy to help or elaborate on any of the points above -- please do reach out!

File: README.md [hunk 1]
```diff
@@ -59,6 +59,8 @@ further to fit a particular use case.
   popular API auth schemes
 - [reCAPTCHA enterprise](references/recaptcha-enterprise) - A reference for
   API protection against bot leveraging reCAPTCHA enterprise
+- [Eventarc + ApiHub integration](references/eventarc-apihub) - A reference for
```
Comment (OmidTahouri) [inline README.md:62]: Move to `## Labs`

File: references/eventarc-apihub/README.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
+# Workflow-based automation for Apigee X using API Hub and EventArc
+
+A common use-case for integrating Apigee with Api-Hub using EventArc and Workflows.
```
Comment (OmidTahouri) [inline references/eventarc-apihub/README.md:3]: Let's get this text body formatted a bit better for readability

File: references/eventarc-apihub/README.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+- [wsdl2apigee](https://github.com/apigee/wsdl2apigee) - An open source project that extends Apigee functionality to provide the ability to expose a SOAP Service as a REST API (over JSON or XML) or
+  expose a SOAP Service in a pass through mode using a WSDL descriptor.
+
+## License
+
+All solutions within this repository are provided under the Apache 2.0 license. Please see the LICENSE file for more detailed terms and conditions.
+
+## Contact
+Questions, issues and comments should be directed to
+[matteo.longo@soft.it](mailto:matteo.longo@soft.it).
```
Comment (OmidTahouri) [inline references/eventarc-apihub/README.md:80]: Please direct readers to raise GitHub issues instead.

**Review by OmidTahouri — CHANGES_REQUESTED** [review]

**Comment by OmidTahouri** [comment]
Closing this ahead of new incoming PR

**Pull request closed without merging**